Manager/BackupManager.php

```diff
@@ -77,28 +77,13 @@ public function execute()
         try {
             // If we catch an exception or not, we would still like to try cleaning up after us
             $this->logger->info('[dizda-backup] Cleaning up after us.');
-            $this->processor->cleanUp();
+            $this->processorManager->cleanUp();
         } catch (IOException $e) {
-            $this->logger->error('[dizda-backup] Cleaning up failed.');
+            $this->logger->error('[dizda-backup] Cleaning up failed.', array('exception' => $e));
 
             return false;
         }
 
         return $successful;
     }
-
-    public function getClientManager()
-    {
-        return $this->cm;
-    }
-
-    public function getDatabaseManager()
-    {
-        return $this->dbm;
-    }
-
-    public function getProcessorManager()
```

Comment:
NP, I'll be glad to explain how/why I use it when it is re-thinked to help find the best solution.